docs/contributing/testing_guide.rst

```diff
@@ -9,6 +9,50 @@ Testing Guide
    :local:
    :backlinks: none
 
+Summary
+=======
+
+* New functionality added to PlasmaPy must also have tests.
+
+* Tests are located in files that begin with :file:`test_` which are
+  inside subdirectories named :file:`tests`.
+
+* Tests are either functions beginning with ``test_`` or classes
+  beginning with ``Test``.
+
+* To install the packages needed to run the tests:
+
+  - Open a terminal.
+
+  - Navigate to the top-level directory (probably named
+    :file:`PlasmaPy`) in your local clone of PlasmaPy's repository.
+
+  - If you are on MacOS or Linux, run:
+
+    .. code-block:: console
+
+       python -m pip install -e.[tests]
+
+    If you are on Windows, run:
+
+    .. code-block:: console
+
+       py -m pip install -e .[tests]
```

Comment:
But it's fine here since Windows is unlikely to have zshell.

Tbh it'd be great to make this a... thing... how do you call it... take a look at https://setuptools.pypa.io/en/latest/userguide/entry_point.html#:~:text=add%20a%20console%20script%20entry%20point%20to%20your%20configuration%3A , just above "After installing the package", how they have separate sort-of-tabs for pyproject.toml, setup.cfg, setup.py? That'd be awesome here.

Reply:
Agreed!  I vaguely recall finding a Sphinx extension that made it fairly straightforward to do that, but I'd have to look to find it again.  I think there'd be a few other places where this would be helpful.

Reply:
I believe it's https://sphinx-tabs.readthedocs.io/en/latest/ :)